src/hooks/use-async-effect.test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import "jest-extended";
```

Comment:
Would you be able to throw this in `setupTests.ts`? I believe that should give you the proper typing for the extended matchers in any test file.

Reply:
Looks like I didn't even need that line at all since it was already in the `setupTests.ts` file.